server/controllers/spreadsheet-controller.js

```diff
@@ -432,6 +433,10 @@ var processGRNmap = function (path, res, app) {
     helpers.attachFileHeaders(res, path);
     network = parseSheet(sheet);
 
+    // Parse expression and 2-column data, then add to network object
+    var additionalData = parseAdditionalSheets(sheet);
+    Object.assign(network, additionalData);
+
     return (network.errors.length === 0) ?
     // If all looks well, return the network with an all clear
```

Comment:
Didn’t see this previously: note lines 441-444 are part of the conditional expression that starts in line 440, so should actually be indented.